RELEASE.md

```diff
@@ -0,0 +1,96 @@
+# Notes on how to publish a Nessie release
+
+## Build Nessie incl Gradle-Plugin and pynessie
+
+This is a prerequisite step, that's repeated below for the "final" release version.
+Run one command from the snippet below at a time - one after the other - not "copy & paste". 
+
+```shell
+# cd to nessie root directory
+
+# Manually verify you're running Java 11
+java -version
+
+# Clean all build files
+./mvnw clean
+(cd tools/apprunner-gradle-plugin/ ; ./gradlew clean)
+
+# Sanity builds
+./mvnw install -Pnative -Pjdk8-tests -Pcode-coverage,jdk8-tests,native
+(cd tools/apprunner-gradle-plugin/ ; ./gradlew build)
+(cd python ; . venv/bin/activate ; python3 setup.py sdist bdist_wheel)
+```
+
+## Actual release
+
+Update the environment variables at the top to your needs, then run
+*ONE COMMAND AT A TIME MANUALLY AFTER THE OTHER !!!*
```

Comment:
If we want people to do this then we shouldn't make it a single code block. I personally will *ALWAYS* copy the entire code bloc and execute it.

Reply:
can parts of this be removed actually? This is being done in actions right?

Reply:
Yea - just leaving it for now to know what was done manually to create a release. Not meant to go into `main`.